### PR TITLE
Story/1088 - Rename packaging module and tidy up module manifests/Python files

### PR DESCRIPTION
**Pull request by PeterC10**
"Advanced" terminology dropped on packaging module, renamed to fit in with other stock modules.

Module manifests updated so all authors now say "Unipart Digital".

Removed utf-8 coding at top of Python files where set, as this is not needed.

### CODE REVIEW

File: addons/edi_notifier/__init__.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```
Comment (kdwyer) [inline addons/edi_notifier/__init__.py:1]: Bit of a nit, but an initial blank line in a file seems wrong somehow (this applies to a number of files)?

**Review by kdwyer — APPROVED** [review]
One nit but LGTM

**Review by LoKKeR96 — APPROVED** [review]
LGTM

**Review by adamrpatrick — APPROVED** [review]

File: addons/edi_notifier/__init__.py (hunk 1 above)
Comment (PeterC10) [reply addons/edi_notifier/__init__.py]: Yes agreed, I used a reg-ex that would remove the line with utf-8, not just empty it. But I didn't consider that some files with this set also left a blank line underneath. Have removed.

**Pull request merged**